### ISSUE TITLE
Application Gateway WAF policy doesn't support operator "any"

### ISSUE THREAD

**Issue by vsmreddy18** [hashicorp/terraform-provider-azurerm#21015]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

Hi Team,

I am trying to create the custom rule in application Gateway WAF policy in the azure portal operator "any " is exist but terraform doesn't support and there is no possible value to support "any" operator

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

](url)

### Terraform Version

0.13

### AzureRM Provider Version

3.9.0

### Affected Resource(s)/Data Source(s)

azurerm_web_application_firewall_policy

### Terraform Configuration Files

```hcl
waf_custom_rules = [
    {
      name      = XXXXXX
      priority  = 1
      rule_type = "MatchRule"
      action    = "Allow"
      match_conditions = [
        {
          match_variables = [
            {
              match_variable = "RequestHeaders"
              selector       = "Host"
            }
          ]

          match_values       = XXXXXX
          operator           = "Equal"
          negation_condition = false
          transforms         = ["Lowercase"]
        },
        {
          match_variables = [
            {
              match_variable = "RequestHeaders"
              selector       = XXXXXX
            }
          ]
          match_values       = XXXXXX
          operator           = "BeginsWith"
          negation_condition = false
          transforms         = []
        }
      ]
```

### Debug Output/Panic Output

```shell
we should required to use operator "any" for custon rules
```

**Comment by ms-zhenhua** [comment]
Hi @vsmreddy18, thank you for reaching out.
Seems this issue is duplicated with https://github.com/hashicorp/terraform-provider-azurerm/issues/19013, which will be resolved in [v3.49.0](https://github.com/hashicorp/terraform-provider-azurerm/milestone/230)

**Comment by mybayern1974** [comment]
Per ms-zhenhua's finding, I'm closing this issue as duplicate. Feel free to suggest to re-open if this issue is talking about a different thing as #19013
